### ISSUE TITLE
Ability to choose a custom background image / color

### ISSUE THREAD

**Issue by iomintz** [SubhamTyagi/Last-Launcher#12]

**Comment by Wilker-a** [comment]
useful settings could include

- background opacity
- background (wallpaper|live wallpaper)
- text (shadows|outline) and background opacity, and color
- maybe an option to make the text background corners rounded

**Comment by alternativesurfer** [comment]
Yes, this would be great because I don't have enough apps to fill my screen....which means I am left with a huge white space at the bottom under the black background that Last Launcher takes up.
